pyproject.toml

```diff
@@ -38,7 +38,7 @@ before-all = [
   "cd ~/",
   "git clone --branch 5.2.1 https://github.com/jemalloc/jemalloc.git",
   "cd jemalloc",
-  "./autogen.sh --disable-cxx --with-jemalloc-prefix=local --with-install-suffix=local",
+  "./autogen.sh --disable-cxx --with-jemalloc-prefix=local --with-install-suffix=local --disable-tls --disable-initial-exec-tls",
```

Comment:
This will disable tls for macos, not linux. Line 72 is where the same setup is declared for linux.